src/Symfony/Component/Messenger/Transport/Serialization/Serializer.php

```diff
@@ -119,6 +127,8 @@ private function decodeStamps(array $encodedEnvelope): array
 
             try {
                 $stamps[] = $this->serializer->deserialize($value, substr($name, \strlen(self::STAMP_HEADER_PREFIX)).'[]', $this->format, $this->context);
+            } catch (NotNormalizableValueException $e) {
```

Comment:
But this is denormalization at this stage, right?

Reply:
True, but it's still a `NotNormalizableValueException` that's thrown there: https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Serializer/Serializer.php#L194